frontend/stepper/api.ts

```diff
@@ -138,11 +142,13 @@ function getNodeStartRow(stepperState: StepperState) {
     return range && range.start.row;
 }
 
-export function makeContext(state: StepperState, interact: Function): StepperContext {
+export function makeContext(stepper: typeof initialStateStepper, interact: Function): StepperContext {
```

Comment:
It would probably be better to define `type Stepper = typeof initialStateStepper`; in `index.ts` and then use that type.